src/Utils/Roles.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Utils;
+
+class Roles
```

Comment:
Enum?

Reply:
What we want as output is a string so this is easier to use I think